app-install-banner/deferred-banner/readme.md

```diff
@@ -1,13 +1,13 @@
-App Install Banner User Response Sample
+Deferred App Install Banner Sample
 ===
 
-See https://googlechrome.github.io/samples/app-install-banner/useraction-banner/index.html for a live demo.
+See https://googlechrome.github.io/samples/app-install-banner/deferred-banner/index.html for a live demo.
 
 
 #### Directions:
 
-* Install Chrome, or another browser that supports Service Workers and manifest
-* Visit the site over two separate days.
-* Or, more usefully, enable chrome://flags/#bypass-app-banner-engagement-checks
+* Install Chrome, or another browser that supports Service Workers and manifest.
+* Wait a few minutes.
```

Comment:
I wonder if it's still true... @dmurph may know

Reply:
You shouldn't need to wait, it should just promote right away if you have the other requirements (display set, start_url set, icon that satisfies the requirement, etc)

Reply:
Also - ServiceWorkers are no longer required in Chrome (maybe also Chromium in general, IDK about Edge) for promotion.

Reply:
Removed references to service workers, the flag, and waiting for the button to be activated